src/main/diagnostics/diagnostics.ts

```diff
@@ -1,13 +1,18 @@
+import Logging from '@/utils/logging';
+
+const console = Logging.diagnostics;
+
 export enum DiagnosticsCategory {
-  Utilities = 'Utilities',
+  Kubernetes = 'Kubernetes',
   Networking = 'Networking',
+  Utilities = 'Utilities',
 }
 
 /**
  * DiagnosticsCheckerResult is the result for running a given diagnostics
  * checker.
  */
-type DiagnosticsCheckerResult = {
+export type DiagnosticsCheckerResult = {
```

Comment:
From an earlier commit, but this type has a mixture of single-star and jsdoc double-star comments. My IDE highlights the `/**...*/` ones, (and greys out the single-star ones), so maybe go with double-star.